composer.json

```diff
@@ -52,7 +52,9 @@
         "squizlabs/php_codesniffer": "3.*",
         "php-parallel-lint/php-parallel-lint": "^1.0",
         "meyfa/phpunit-assert-gd": "^2.0.0|^3.0.0",
-        "dms/phpunit-arraysubset-asserts": "^0.1.0|^0.2.1"
+        "dms/phpunit-arraysubset-asserts": "^0.1.0|^0.2.1",
+        "nunomaduro/larastan": "^1.0",
+        "orchestra/testbench": "dev-master"
```

Comment:
Why was testbench added here?

Reply:
It's needed for testing Laravel modules and not "true" applications - which Storm kinda sorta is not. I found I needed it to even get PHPStan working in the first place.